app/twodii_datawarehouse/file_import/df_utils.py

```diff
@@ -0,0 +1,111 @@
+"""Utility functions for operating on dataframes for file import."""
+import pandas as pd
+
+
+def find_header_row(
```

Comment:
Have you tested how well this works on our current files? Not a huge priority since it will return FALSE if it can't find a header, but still curious

Reply:
I did test this on (some) current GD files. THey're actually what caused me to write it in the first place.